packages/espree/package.json

```
@@ -56,7 +59,9 @@
     "acorn"
   ],
   "scripts": {
-    "build": "rollup -c rollup.config.js",
+    "attw": "attw --pack",
```

In the `eslint` repo, we use `lint:types` as a script name to run `attw`. I'd suggest doing the same here for consistency.

```suggestion
    "lint:types": "attw --pack",
```